packages/lesswrong/components/sunshineDashboard/UsersReviewInfoCard.tsx

```diff
@@ -191,11 +194,12 @@ const UsersReviewInfoCard = ({ user, refetch, currentUser, classes }: {
         {showReviewTrigger && <MetaInfo className={classes.legacyReviewTrigger}>{reviewTrigger}</MetaInfo>}
       </div>
       <MetaInfo className={classes.referrerLandingPage}>
-        {user.associatedClientId?.firstSeenReferrer && <div>Initial referrer: {user.associatedClientId.firstSeenReferrer}</div>}
+        {firstClientId?.firstSeenReferrer && <div>Initial referrer: {firstClientId.firstSeenReferrer}</div>}
```

Comment:
Same here.

Reply:
This one appears to have been missed in the previous PR that attempted to refactor this block out. Refactored.